GameEngine/Commands/MoveCommand.cs

```diff
@@ -18,9 +18,10 @@ public void Exceute(EngineInternal engine)
             placeToMove.Add("Cancel");
             var placeToMoveTo = Console.Choose("Where would you like to move to?", placeToMove);
 
-            if (placeToMove.Equals("Cancel"))
+            if (placeToMoveTo.Equals("Cancel"))
             {
                 Console.WriteLine("Canceled Move");
+                GameState.CurrentGameState.CharacterLocations["Player"] = playerLoc;
```

Comment:
It doesn't seem like we need to set it to what it's already set to. That'd be like writing a = a;